homeassistant/components/climate/myicomfort.py

```diff
@@ -0,0 +1,225 @@
+"""
+Support for the Lennox iComfort WiFi thermostat.
+
+Sample entry for configuration.yaml:
+
+climate:
+  - platform: myicomfort
+    name: Downstairs
+    username: YOUR_MYICOMFORT_USERNAME
+    password: YOUR_MYICOMFORY_PASSWORD
+
+"""
+
+import logging
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
```

Comment:
Remove this blank line.

Reply:
Do I need to remove the blank line before, or after,  import homeassistant.helper, or both?

Reply:
Remove it at line 18. The review comment is always directed at the bottom line in the code box shown.

We follow PEP8.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I thought it might be directed at the last line but wasn't sure.  Thanks for helping me learn the ropes, this is my first ever PR so I will probably have more newbie questions along the way.

I read thru that section of PEP8 and understand how the splits should be now.  I will get that fixed.